### PR TITLE
More verbose test errors

### PR DESCRIPTION
**Pull request by bwatkinson**
#23 should be merged before this. However, this should help us in figuring out test failures in the future.

### CODE REVIEW

File: tests/common.sh [hunk 1]
```diff
@@ -47,22 +47,31 @@ initialize_test() {
...
+  local message=$@
 
   local rc=1
   if [ $status -eq 0 ]; then
       pass
       rc=0
   elif [ $status -eq -1 ]; then
-      skip
-      rc=0
+      skip $message
```
Comment (calccrypto) [inline tests/common.sh:58]: `"${message}"`

**Comment by calccrypto** [comment]
Should shellcheck be run in the CI?

https://github.com/mar-file-system/GUFI/blob/master/.github/workflows/check.yml#L83

File: tests/common.sh (hunk 1 above)
Comment (bwatkinson) [reply tests/common.sh:58]: I just tested doing this, and there is no output from the `fail` function by doing this.

Comment (calccrypto) [reply tests/common.sh:58]: weird

**Comment by bwatkinson** [comment]
> Should shellcheck be run in the CI?
> 
> https://github.com/mar-file-system/GUFI/blob/master/.github/workflows/check.yml#L83

If you want to we can. Feel free to open a PR if you want.

**Review by calccrypto — APPROVED** [review]

**Review by MigeljanImeri — APPROVED** [review]

**Pull request merged**